$NetBSD: patch-Configure,v 1.3 2022/10/29 18:28:06 nros Exp $

 * Don't refer to gzip and perl in the tools directory

--- Configure.orig	2016-02-25 14:12:36.000000000 +0000
+++ Configure
@@ -39,9 +39,9 @@ echo "Configuring pari-$pari_release_ver
 . ./get_PATH
 #  We might need the following :
 echo Looking for some tools first ...
-_tools_list='gzip cc gcc ld perl zcat'
+_tools_list='cc gcc ld perl zcat'
 pathspace=`echo $PATH | sed -e "s/$dir_sep/ /g" | sed -e 's,\\\\,/,g'`
-
+echo ..."gzip is $gzip"
 for file in $_tools_list; do
   x=`./locate $file '' $pathspace`
   eval $file=$x
@@ -53,6 +53,7 @@ for file in $_tools_list; do
 done
 if test -z "$zcat" -a -n "$gzip"; then zcat="$gzip -dc"; fi
 
+perl=$PERL5
 ####################### CONFIGURE - ARCHITECTURE ############################
 . ./get_archos # arch, osname
 config_log="$TOP/config-$arch-$osname$$.log"
@@ -119,14 +120,14 @@ fi
 . ./get_static  # static
 # For dynamic linking, before and after installing
 runpath=\"$libdir\"
-LDDYN="-lpari"
+LDDYN="-Wl,-R$libdir -L$libdir -lpari"
 # get_modld needs $includedir from get_install, static, and LDDYN
 . ./get_modld   # $_modld_list
 
 # Which copy, SHELL ?
 case "$osname" in
   os2) ln_s=cp;      make_sh=sh;;
-  *)   ln_s="ln -s"; make_sh="/bin/sh";;
+  *)   ln_s="ln -sf"; make_sh="/bin/sh";;
 esac
 ####################### CONFIGURE - CLEANUP #################################
 rm -f gmon.out # created by Configure -pg
